var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -174,3 +213,43 @@ def configure_args(self):
         args.append('--with-dri-drivers=' + ','.join(args_dri_drivers))
 
         return args
+
+    @property
+    def gl_libs(self):
```

Comment:
Look for `libOpenGL` from `libglvnd` when `+glvnd`, `libGL` from `self` otherwise.

Also don't need to bother with the second`if` guard here since the `gl` virtual package is only provided when `+opengl`.  Only one library needs to be r